src/sections/OurDoctors/TeamSlider.jsx

```diff
@@ -0,0 +1,108 @@
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import image1 from "../../assets/img_01.webp";
+import image2 from "../../assets/img_02.webp";
+import image3 from "../../assets/img_03.webp";
+import image4 from "../../assets/img_04.webp";
+import image5 from "../../assets/img_05.webp";
+import PropTypes from "prop-types";
+import StarRating from "./StarRating";
+
+const teamMembers = [
+  {
+    name: "Jubayer Al Hasan",
+    role: "Marketing Expert",
+    imageUrl: image1,
+    rating: 4,
+  },
+  {
+    name: "Jannatul Ferdaus",
+    role: "Broker",
+    imageUrl: image2,
+    rating: 5,
+  },
+  {
+    name: "Chris Matial",
+    role: "Broker",
+    imageUrl: image3,
+    rating: 3,
+  },
+  {
+    name: "Mark Filo",
+    role: "CEO & Founder",
+    imageUrl: image4,
+    rating: 5,
+  },
+  {
+    name: "Professor Dr D. Filo",
+    role: "CEO & Founder",
+    imageUrl: image5,
+    rating: 4,
+  },
+];
+
+const TeamSlider = () => {
```

Comment:
This should be doctor slider.

Also add this to `containers` and not `sections`
Also add storyboook item for this.

Reply:
Resolved. thanks!